avssync/metrics.go

```diff
@@ -17,28 +17,55 @@ const (
 	UpdateStakeStatusSucceed UpdateStakeStatus = "succeed"
 )
 
-var (
-	updateStakeAttempt = promauto.NewCounterVec(prometheus.CounterOpts{
-		Namespace: metricsNamespace,
-		Name:      "update_stake_attempt",
-		Help:      "Result from an update stake attempt. Either succeed or error (either tx was mined but reverted, or failed to get processed by chain).",
-	}, []string{"status", "quorum"})
-	txRevertedTotal = promauto.NewCounter(prometheus.CounterOpts{
-		Namespace: metricsNamespace,
-		Name:      "tx_reverted_total",
-		Help:      "The total number of transactions that made it onchain but reverted (most likely because out of gas)",
-	})
-	operatorsUpdated = promauto.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: metricsNamespace,
-		Name:      "operators_updated",
-		Help:      "The total number of operators updated (during the last quorum sync)",
-	}, []string{"quorum"})
-)
+type Metrics struct {
+	UpdateStakeAttempts *prometheus.CounterVec
+	TxRevertedTotal     prometheus.Counter
+	OperatorsUpdated    *prometheus.GaugeVec
```

Comment:
make these private to force using the setter functions? This would lead to better code consistency in case some later programmer comes in and sets them by hand